lib/ueberauth/strategy/amco/api/access_token.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Ueberauth.Strategy.Amco.API.AccessToken do
+  alias OAuth2.Response
+  alias Ueberauth.Strategy.Amco.OAuth
+
+  def authorize(token) do
+    OAuth.authorize_access_token(token)
+    |> process_response()
+  end
+
+  defp process_response({:ok, %Response{status_code: 200} = response}) do
+    case response.body do
+      %{"active" => true} = data -> {:ok, data}
```

Comment:
This is the same as `%{"active" => true} -> {:ok, response.body}`

IMO data is a very broad  term, so if you want to express the kind of response you are setting, maybe look for a different word, or just leave the response.body  as the returned data?

Reply:
Fixed, thanks.